mmrazor/models/mutators/channel_mutator/channel_mutator.py

```diff
@@ -168,6 +169,23 @@ def traverse(module, prefix):
 
         traverse(supernet, '')
 
+    def bind_mask_type(self, supernet: Module):
```

Comment:
The doc-string seems to be the same as method `bind_mutable_type`. And the name of `bind_mask_type` is a bit confusing, since it binds a name to mutable but not a type.

Reply:
mutable type主要是为了debug以及显示，如果没有type，打印出来的conv/linear对应的mutable不知道是in还是out，不好看。
想的是一个mutable的type就是dynamic layer的mutable in / out这些，或者是mutable num_feature这种，为了保证灵活性，没把type设成in/out这种。dynamic layer里面具有的mutable的名字，就可以当做mutable type。